iphone/cli/commands/_build.js

```diff
@@ -5818,7 +5818,7 @@ iOSBuilder.prototype.copyResources = function copyResources(next) {
 					this.logger.info(__('Processing JavaScript files'));
 					const sdkCommonFolder = path.join(this.titaniumSdkPath, 'common', 'Resources');
 
-					async.eachSeries(Object.keys(jsFiles), function (file, next) {
+					async.each(Object.keys(jsFiles), function (file, next) {
 						setImmediate(function () {
```

Comment:
Is this `setImmediate` necessary anymore?

Reply:
I don't remember why the `setImmediate()` was there in to begin with in this situation, however I generally use `setImmediate()` to guarantee a block is truly async (which is sometimes necessary if returning an event emitter or there's unified error handling) or to collapse the call stack, though this function isn't very recursive.